changelog.md

```diff
@@ -7,6 +7,13 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ----
 
+## [v2.1.1] => 2022-JUN-3
```

Comment:
Since this is a new feature, it should be version 2.2 :)

Reply:
Tests added, however, they do not have a test API so I instead checked to make sure the required keys were provided in the payload